src/Sdk/DTLogging/Logging/SecretMasker.cs

```diff
@@ -40,6 +40,19 @@ private SecretMasker(SecretMasker copy)
             }
         }
 
+        /// <summary>
+        /// Provide callers with a recommendation on what to consider a secret.
+        /// This is helpful in cases where JSON (for example) is broken into multiple lines
+        /// and we don't want to start treating standalone JSON control characters as secrets.
+        /// </summary>
+        public int DerivedSecretRecommendedMinimumLength
```

Comment:
i would just define this on some file not under `src/SDK` since those better deployed with the server code first.